BlogTemplate/Pages/AlreadyRegistered.cshtml

```diff
@@ -0,0 +1,6 @@
+@page
+@{
+    ViewData["Title"] = "Already Registered";
+}
+
+<h1>You are already logged in.</h1>

```

Comment:
This page shows up when you go to /account/register and try to register a new user. I think it would make more sense to say "You are already registered."